demos/tutorials/Deconvolution1D.py

```diff
@@ -0,0 +1,164 @@
+"""
```

Comment:
As with previous howto, I suggest a filename that does not coincide with the class name. Maybe prepend or append "Tutorial" or something similar?